README.md

```diff
@@ -1 +1,52 @@
-# Quran-Apies
\ No newline at end of file
+# Quran-Apies
+## With Al-Qura'an Al-Kareem (مع القرآن الكريم)
+This app provides three user services: 
+    1- General information about Soura (chapters) of Al-Qura'an Al-Kareem.
+    2- Playing audio for specific reciter for certain Soura of Al-Qura'an Al-Kareem.
+    3- Search in the entire Quara'an for the exisitance of certain word.
+
+### The live Dem   (مع القرآن الكريم)
+You are more than welcome to visit our app here ( for github pages)
+### The problem this app solve:
+Many times, muslims user needs to access quickly general information about Sour Al-Qura'an Al-Kareem, listening to recitation of Soura of Al-Qura'an Al-Kareem, or finding arabic word in Al-Qura'an Al-Kareem for memorization or learning purposes. Thus, we were inspired to build a simple to use app for Al-Qura'an Al-Kareem. 
+### App wireframe
+The design of the app were done through figma. It is based to simplfy user experience and make it as peaceful and smooth as possible.
+<img src="/assets/wireframe.png" >
+
+### User journey
+The muslim user is welcomed by a home page. Home page shows the user three button choices: 
+- Detailed informatin of Soura (تفاصيل عن سور).
+- Listening to Soura (الاستماع لسورة).
+- Search for word in Al-Qura'an Al-Kareem (ابحث في كلمات القرآن الكريم).
+When the user click on one of these button, it will land to specific page fullfilling the descriped function of the button. 
+### User story
+As a muslim user, I am welcomed to this app at home page. I can see three buttons which are 
+- Detailed informatin of Soura (تفاصيل عن سور).
+- Listening to Soura (الاستماع لسورة)
+- Search for word in Al-Qura'an Al-Kareem (ابحث في كلمات القرآن الكريم)
+As a muslim user, I can press on "Detailed informatin of Soura (تفاصيل عن سورة)" button which leads me to Soura information page containing a search form. In this form, I can type Soura name and press (استخرج) button. Then general information of this Soura name will be listed. These information are the squence number of the Soura, the verses count, and the place of revelation.
+I can press on Listening to Soura (الاستماع لسورة) button which will leads me to listening page containing two scroll down manus. The first menu allow me to choose Soura I want to listen to. The second menue allow me to choose the name of reciter I like to hear his recitation. Then, an audio time-line will play. I can stop it or make it continue if I press arrow button.  
+I can press on Search for word in Al-Qura'an Al-Kareem (ابحث في كلمات القرآن الكريم) button which leads me to word search page. In this page, I can enter an arabic word to find its presences in Sour of Al-Quara'an Al-kareem. I can see the number of results, and all Sours that contain the word I searched for.
+
```

Comment:
**here at user story try to make a list instead of a paragraph for the things that a user can do at your app.**